test/answer.e2e-spec.ts

```diff
@@ -149,60 +151,122 @@ describe('Answers Module', () => {
         expect(respond.body.data.id).toBeDefined();
         questionId.push(respond.body.data.id);
       }
+
       await createQuestion(
         '我这个哥德巴赫猜想的证明对吗？',
         '哥德巴赫猜想又名1+1=2，而显然1+1=2是成立的，所以哥德巴赫猜想是成立的。',
       );
+      await createQuestion('求助', '给指导老师分配了任务，老师不干活怎么办？');
+      await createQuestion('提问', '应该给指导老师分配什么任务啊');
+      await createQuestion('不懂就问', '忘记给指导老师分配任务了怎么办');
+      await createQuestion('小创求捞', '副教授职称，靠谱不鸽，求本科生带飞');
+      await createQuestion('大创', '极限捞人');
     });
-
-    it('should create an auxiliary user', async () => {
+    it('should create some auxiliary users', async () => {
       [auxUserId, auxAccessToken] = await createAuxiliaryUser();
```

Comment:
why not conbine into foreach?

Reply:
If combine it into foreach the userList will be regarded as undefined.